x-pack/plugins/security_solution/public/cases/pages/helpers.ts

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
Why the helpers moved here?

Reply:
because almost all the `x-pack/plugins/security_solution/public/cases/components/*` will be removed as it won't be needed anymore. This helper is now used by `x-pack/plugins/security_solution/public/cases/pages/index.tsx`